evm/src/util.rs

```diff
@@ -45,6 +47,29 @@ pub fn trace_rows_to_poly_values<F: Field, const COLUMNS: usize>(
         .collect()
 }
 
+/// Returns the lowest LE 32-bit limb of a `U256` as a field element,
+/// and errors if the integer is actually greater.
+pub(crate) fn u256_lowest_limb<F: Field>(u256: U256) -> Result<F, ProgramError> {
```

Comment:
I feel like this function name seems to imply that it would take the lowest limb of any U256, not just U256s that entirely fit in one limb?

Reply:
I think `u256_to_u32` or something similar would be clearer

Reply:
It's true the correct naming is confusing, I'll address that. Same for the u64 version.